### PR TITLE
corerouter: use patched version of bird2 and add proper inbound metrics

### PR DESCRIPTION
**Pull request by spolack**
please test

### CODE REVIEW

File: roles/cfg_openwrt/templates/corerouter/bird.conf.j2 [hunk 1]
```diff
@@ -45,14 +60,12 @@ protocol babel {
   {% set default_mesh_metric = hostvars[inventory_hostname].get('mesh_metric_adhoc_' ~ nw.get('mesh_radio'), mesh_metric_default_in) %}
```
Comment (Noki) [inline roles/cfg_openwrt/templates/corerouter/bird.conf.j2:60]: the comment should be moved up and the line with `set default_mesh_metric` should be removed as it does not do anything here anymore

**Review by Noki — CHANGES_REQUESTED** [review]
works for me even without the requested changes.

**Review by PolynomialDivision — APPROVED** [review]

**Comment by PolynomialDivision** [comment]
oh wait, please add something to commit message

**Comment by PolynomialDivision** [comment]
Please add

```
Link: https://bird.network.cz/pipermail/bird-users/2023-February/016705.html
```

**Review by Noki — APPROVED** [review]

**Pull request merged**